ledger/ledgercore/statedelta.go

```diff
@@ -88,6 +88,8 @@ type KvValueDelta struct {
 // If adding a new field not explicitly allocated by PopulateStateDelta, make sure to reset
 // it in .ReuseStateDelta to avoid dirty memory errors.
 // If adding fields make sure to add them to the .Reset() method to avoid dirty state
+//
+//msgp:ignore StateDelta
```

Comment:
This will prevent us from ever msgp-encoding this data structure right? Not necessarily a bad thing, just want to be clear.

Reply:
I believe this is only for the pre-generated Marshal/Unmarshal functions. We're still able to use the reflection based msgp encoding functions.

Reply:
Right, the codegen which might be used when sending these messages over the network for example.